package.json

```diff
@@ -1,26 +1,37 @@
 {
   "name": "leaderboard_joserm",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack --config webpack.config.js --watch"
   },
```

Comment:
- [ ] [OPTIONAL]
Let's modify the _script_ section of the `package.json` file to ensure that the app can be automatically launched in the browser when the command `npm start` is run.

```javascript
 "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack",
    "start": "webpack serve --open"
  },
```